sdk: generalize for any SDKs and enhance explanations (#8232)
tldr-pages/tldr: pages/common/sdk.md

> Supports Java, Groovy, Scala, Kotlin, Gradle, Maven, Vert.x and many others.
> More information: <https://sdkman.io/usage>.

> - Install a SDK version:

Reviewer: ```suggestion
- Install an SDK version:
```
(https://oneminuteenglish.org/en/a-or-an-before-acronyms/)

But I think even better would be
```suggestion
- Install a specific SDK version:
```

`sdk install {{sdk_name}} {{sdk_version}}`

> - Use a specific SDK version in the current terminal:

Reviewer: ```suggestion
- Use a specific SDK version for the current terminal session:
```

`sdk use {{sdk_name}} {{sdk_version}}`

> - Show the stable version of a SDK:

Reviewer: ```suggestion
- Show the stable version of the current SDK:
```
Not sure if this is correct

Author: I think it's not correct. :smile_cat:

Reviewer: Then maybe
```suggestion
- Show the stable version of any available SDK:
```

`sdk current {{sdk_name}}`

> - Show the stable versions of installed SDKs:

Reviewer: ```suggestion
- Show the stable versions of all installed SDKs:
```

`sdk current`

> - List all SDKs:

Reviewer: Installed or available?

Reviewer: ```suggestion
- List all available SDKs:
```

`sdk list`

> - List all versions of a SDK:

Reviewer: Again, installed or available? Also:
```suggestion
- List all versions of an SDK:
```

`sdk list {{sdk_name}}`

> - Upgrade a SDK to the latest stable version:

Reviewer: ```suggestion
- Upgrade an SDK to the latest stable version:
```

`sdk upgrade {{sdk_name}}`

> - Uninstall a SDK version:

Reviewer: ```suggestion
- Uninstall a specific SDK version:
```
